src/DynamoCoreWpf/ViewModels/Search/NodeSearchElementViewModel.cs

```diff
@@ -101,6 +101,8 @@ public bool HasDescription
             get { return (!string.IsNullOrEmpty(Model.Description)); }
         }
 
+        public NodeCategoryViewModel Category { get; set; }
+
```

Comment:
Because of places where `NodeSearchElementViewModel` is created looks like it is not possible.

We are creating instance of this type in `SearchViewModel.MakeNodeSearchElementVM`. This method is used as Func<T> parameter in `Select`. For example: in method `SearchViewModel.CategorizeEntries` we have a line
```C#
var category = new NodeCategoryViewModel(name, es.OrderBy(en =>
    en.Name).Select(MakeNodeSearchElementVM), subs);
```
If we add to `MakeNodeSearchElementVM` even default argument it will not compile. Other option rewrite it and all places where we have such construction.

Let me know if I should do this.